`generate_cached_results` is failing in mteb-results, blocking leaderboard updates

### Describe the bug

I noticed that in the latest runs of the mteb-results repository, all generate_cached_results workflows are failing, which prevents the leaderboard from being updated.

<img width="1646" height="567" alt="Image" src="https://github.com/user-attachments/assets/e0df171c-2efc-4a7e-b573-1aeccac5097e" />

```
Run # Check if __cached_results.json.gz was created
📦 Generated cache file: 85.2MB
📋 Switching to existing cached-data branch
Downloading __cached_results.json.gz (89 MB)
Error downloading object: __cached_results.json.gz (4d8af65): Smudge error: Error downloading __cached_results.json.gz (4d8af65fe86b331510cf6e7a94f752f06e869c8a0b6d2aef6f84690996ce3c7f): batch response: This repository exceeded its LFS budget. The account responsible for the budget should increase it to restore access.

Errors logged to '/home/runner/work/results/results/.git/lfs/logs/20260110T151722.806170654.log'.
Use `git lfs logs last` to view the log.
error: external filter 'git-lfs filter-process' failed
fatal: __cached_results.json.gz: smudge filter lfs failed
Error: Process completed with exit code 128.
```

Could you please:
- Fix the failure in generate_cached_results in the mteb-results repository
- After it is fixed, manually re-trigger:
  - generate_cached_results in mteb-results
  - leaderboard_refresh in the mteb repository
- Update the leaderboard to reflect the latest results

Thanks a lot for maintaining MTEB — this issue is currently blocking new results from appearing on the leaderboard, so a fix would be greatly appreciated!


### To reproduce

```
Run # Check if __cached_results.json.gz was created
📦 Generated cache file: 85.2MB
📋 Switching to existing cached-data branch
Downloading __cached_results.json.gz (89 MB)
Error downloading object: __cached_results.json.gz (4d8af65): Smudge error: Error downloading __cached_results.json.gz (4d8af65fe86b331510cf6e7a94f752f06e869c8a0b6d2aef6f84690996ce3c7f): batch response: This repository exceeded its LFS budget. The account responsible for the budget should increase it to restore access.

Errors logged to '/home/runner/work/results/results/.git/lfs/logs/20260110T151722.806170654.log'.
Use `git lfs logs last` to view the log.
error: external filter 'git-lfs filter-process' failed
fatal: __cached_results.json.gz: smudge filter lfs failed
Error: Process completed with exit code 128.
```

### Additional information

_No response_

### Are you interested to contribute a fix for this bug?

No

--- a/.github/workflows/generate_cached_results.yml
+++ b/.github/workflows/generate_cached_results.yml
@@ -22,7 +22,6 @@ jobs:
         with:
           fetch-depth: 0
           token: ${{ secrets.GITHUB_TOKEN }}
-          lfs: true
 
       - name: Setup Python
         uses: actions/setup-python@v5
@@ -39,14 +38,6 @@ jobs:
           python scripts/generate_cached_results.py
         env:
           PYTHONUNBUFFERED: 1
-
-      - name: Install Git LFS
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y git-lfs
-          git lfs install
-          # Explicitly pull LFS files to work around checkout action issues
-          git lfs pull || true
 
       - name: Configure Git
         run: |
@@ -75,8 +66,6 @@ jobs:
             echo "📋 Switching to existing cached-data branch"
             git checkout cached-data
             git pull origin cached-data
-            # Ensure LFS files are available after branch checkout
-            git lfs pull || true
           else
             echo "🆕 Creating new cached-data branch"
             git checkout --orphan cached-data
@@ -89,30 +78,20 @@ jobs:
             mv __cached_results.json.gz.tmp __cached_results.json.gz
           fi
 
-          # Setup Git LFS tracking for the cache file (if not already tracked)
-          if ! git lfs ls-files | grep -q "__cached_results.json.gz"; then
-            git lfs track "__cached_results.json.gz"
-          fi
-
           # Ensure we only have the files we want
-          # Remove all tracked files except README.md and .gitattributes
+          # Remove all tracked files except README.md
           if [ -f "README.md" ]; then
-            git ls-files | grep -v "README.md" | grep -v ".gitattributes" | xargs -r git rm 2>/dev/null || true
+            git ls-files | grep -v "README.md" | xargs -r git rm 2>/dev/null || true
           else
-            git ls-files | grep -v ".gitattributes" | xargs -r git rm 2>/dev/null || true
+            git ls-files | xargs -r git rm 2>/dev/null || true
           fi
 
-          # Add the cached results file (will be tracked by LFS)
+          # Add only the cached results file
           git add __cached_results.json.gz
 
           # Preserve README.md if it exists in the working directory
           if [ -f "README.md" ]; then
             git add README.md
-          fi
-
-          # Add .gitattributes file (required for LFS tracking)
-          if [ -f ".gitattributes" ]; then
-            git add .gitattributes
           fi
 
           # Check if there are changes to commit
@@ -122,7 +101,7 @@ jobs:
             # Verify we're not committing too many files (safety check)
             STAGED_FILES=$(git diff --staged --name-only | wc -l)
             if [ "$STAGED_FILES" -gt 10 ]; then
-              echo "❌ ERROR: Too many files staged ($STAGED_FILES). Expected only 1-3 files (__cached_results.json.gz, README.md, .gitattributes)."
+              echo "❌ ERROR: Too many files staged ($STAGED_FILES). Expected only 1-2 files."
               echo "Staged files:"
               git diff --staged --name-only
               exit 1
@@ -133,7 +112,7 @@ jobs:
             COMMIT_MSG="Update cached results - $TIMESTAMP ($(echo "scale=1; $FILE_SIZE/1024/1024" | bc -l)MB)"
             git commit -m "$COMMIT_MSG"
 
-            # Push to remote (LFS will handle the large file automatically)
+            # Push to remote
             git push origin cached-data
             echo "✅ Successfully updated cached-data branch"
           fi

